packages/inter-protocol/test/psm/gov-replace-committee.js

```diff
@@ -178,57 +245,168 @@ const startNewEconomicCommittee = async ({
   // NB: committee must only publish what it intended to be public
   const marshaller = await E(board).getPublishingMarshaller();
 
-  const { creatorFacet } = await E(zoe).startInstance(
-    committee, // aka electorate
+  trace('Starting new EC Committee Instance');
+  const { instance, creatorFacet } = await E(zoe).startInstance(
+    committee,
     {},
     { committeeName, committeeSize },
     {
       storageNode,
       marshaller,
     },
+    'economicCommittee',
   );
+  trace('Started new EC Committee Instance Successfully');
 
-  const newPoserInvitationP = E(creatorFacet).getPoserInvitation();
+  economicCommittee.reset();
+  economicCommittee.resolve(instance);
 
-  // reset because it's already been resolved
   economicCommitteeCreatorFacet.reset();
   economicCommitteeCreatorFacet.resolve(creatorFacet);
-  return newPoserInvitationP;
 };
 harden(startNewEconomicCommittee);
 
-const main = async permittedPowers => {
-  console.log('starting new economic committee:', runConfig);
-  const newElectoratePoser = await startNewEconomicCommittee(permittedPowers);
+const startNewEconCharter = async ({
+  consume: { zoe },
+  produce: { econCharterKit },
+  installation: {
+    consume: { binaryVoteCounter: counterP, econCommitteeCharter: installP },
+  },
+  instance: {
+    produce: { econCommitteeCharter: instanceP },
+  },
+}) => {
+  const [charterInstall, counterInstall] = await Promise.all([
+    installP,
+    counterP,
+  ]);
+  const terms = await harden({
+    binaryVoteCounterInstallation: counterInstall,
+  });
 
-  /*
-   * put the new economic committee into agoricNames
-   */
+  trace('Starting new EC Charter Instance');
+  const startResult = await E(zoe).startInstance(
+    charterInstall,
+    undefined,
+    terms,
+    undefined,
+    'econCommitteeCharter',
+  );
+  trace('Started new EC Charter Instance Successfully');
+
+  instanceP.reset();
+  instanceP.resolve(E.get(startResult).instance);
+
+  econCharterKit.reset();
+  econCharterKit.resolve(startResult);
+};
+harden(startNewEconCharter);
+
+const addGovernorsToEconCharter = async ({
+  consume: {
+    reserveKit,
+    vaultFactoryKit,
+    econCharterKit,
+    auctioneerKit,
+    psmKit,
+  },
+  instance: {
+    consume: { reserve, VaultFactory, auctioneer },
+  },
+}) => {
+  const { creatorFacet } = E.get(econCharterKit);
+
+  const psmKitMap = await psmKit;
+
+  for (const obj of psmKitMap.values()) {
+    await E(creatorFacet).addInstance(
+      obj.psm,
+      obj.psmGovernorCreatorFacet,
+      obj.label,
+    );
+  }
```

Comment:
Is there any reason not to do this?
```suggestion
  for (const { psm, psmGovernorCreatorFacet, label } of psmKitMap.values()) {
    E(creatorFacet).addInstance(psm, psmGovernorCreatorFacet, label);

    return { publicFacet, creatorFacet };
};
```